addOns/pscanrulesBeta/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## [20] - 2019-07-06
+
+- Tweak InformationDisclosureSuspiciousComments scanner to ignore whitespace before/after suspicious comments terms in the suspicious-comments.txt config file.
```

Comment:
Better to use the name of the scanner instead of the class, that's what's shown to the user mainly.

Reply:
I thought InformationDisclosureSuspiciousComments is the name of the scanner. I don't understand this comment.

Reply:
Whatever the getName function returns is what he means.

Reply:
https://github.com/JonathonWichers/zap-extensions/blob/d424fef78733d08594f59156abd43cdb27379ef4/addOns/pscanrulesBeta/src/main/resources/org/zaproxy/zap/extension/pscanrulesBeta/resources/Messages.properties#L35